### DISCUSSION TITLE
Deployment Failure

### DISCUSSION THREAD

**Discussion by JusticeSol** [Cyfrin/foundry-full-course-cu#54] [Q&A] [answered]
I keep getting this error message when trying to deploy SimpleStorage contract to sepolia testnet - "creation of SimpleStorage errored: [TIMEOUT] Timeout for call sendAsync from udapp"

Any help please?

**Comment by AnouarBF** [comment, marked as answer]
Hello @HybornCV. I also encountered this problem before, just make sure that the rpc url and the private key are correct and match with the one in Alchemy and metamask. also make sure whenever you modify .env file. to type ```source .env``` in the terminal.
Hope that would help!

**Reply by cromewar** [reply]
Yes this is most likely the solution, please let us know if this works for you @HybornCV 
thanks @AnouarBF
